### PR TITLE
2020.12.0 release

### PR DESCRIPTION
**Pull request by jrbourbeau**
This PR adds a changelog entry for the 2020.12.0 release. Since it's a large release I've added a "Highlights" section and it would be good to get another maintainer to briefly review to have a second opinion. cc @jsignell @mrocklin

### CODE REVIEW

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -1,6 +1,133 @@
...
+- Fix sphinx ``currentmodule`` usage (:pr:`6839`) `James Bourbeau`_
+- Fix minimum dependencies CI builds (:pr:`6838`) `James Bourbeau`_
+- Avoid graph materialization during ``Blockwise`` culling (:pr:`6815`) `Richard (Rick) Zamora`_
+- Fixed typo (:pr:`6834`) `Devanshu Desai`_
+- Use ``HighLevelGraph.merge`` in ``collections_to_dsk`` (:pr:`6836`) `Mads R. B. Kristensen`_
+- Respect ``dtype`` in svd ``compression_matrix`` #2849 (:pr:`6802`) `RogerMoens`_
+- Add blocksize to task name (:pr:`6818`) `Julia Signell`_
+- Check for all-NaN partitions (:pr:`6821`) `Rockwell Weiner`_
+- Change "institutional" SQL doc section to point to main SQL doc (:pr:`6823`) `Martin Durant`_
+- Fix #6775 (:pr:`6809`) `David Katz`_
```
Comment (jsignell) [inline docs/source/changelog.rst:84]:
```suggestion
- Fix: DataFrame.join doesn't accept Series as other (:pr:`6809`) `David Katz`_
```

File: docs/source/changelog.rst [hunk 2]
```diff
@@ -1,6 +1,133 @@
...
+All changes
+^^^^^^^^^^^
+
+- Make ``observed`` kwarg optional (:pr:`6952`) `Julia Signell`_
+- Min supported pandas 0.25.0 numpy 1.15.1 (:pr:`6895`) `Julia Signell`_
+- Make order of categoricals unambiguous (:pr:`6949`) `Julia Signell`_
+- Improve "pyarrow-dataset" statistics performance for ``read_parquet`` (:pr:`6918`) `Richard (Rick) Zamora`_
+- Add ``observed`` keyword to ``groupby`` (:pr:`6854`) `Julia Signell`_
+- Make sure ``include_path_column`` works when there are multiple partitions per file (:pr:`6911`) `Julia Signell`_
+- Fix for github issue 6849 (:pr:`6909`) `GFleishman`_
```
Comment (jsignell) [inline docs/source/changelog.rst:31]:
```suggestion
- Fix: array.overlap and array.map_overlap block sizes are incorrect when depth is an unsigned bit type (:pr:`6909`) `GFleishman`_
```

File: docs/source/changelog.rst [hunk 3]
```diff
@@ -1,6 +1,133 @@
...
+- Allow ``pyarrow >2.0.0`` (:pr:`6772`) `Richard (Rick) Zamora`_
+- Support ``pyarrow.dataset`` API for ``read_parquet`` (:pr:`6534`) `Richard (Rick) Zamora`_
+- Add more informative error message to ``da.coarsen`` when coarsening factors do not divide shape (:pr:`6908`) `Davis Bennett`_
+- Only run the cron CI on ``dask/dask`` not forks (:pr:`6905`) `Jacob Tomlinson`_
+- Add ``annotations`` to ``ShuffleLayers`` (:pr:`6913`) `Matthew Rocklin`_
+- Temporarily xfail ``test_from_s3`` (:pr:`6915`) `James Bourbeau`_
+- Added dataframe ``skew`` method (:pr:`6881`) `Jan Borchmann`_
+- Fix ``dtype`` in array ``meta`` (:pr:`6893`) `Julia Signell`_
+- Missing ``name`` arg in ``helm install ...`` (:pr:`6903`) `Ruben van de Geer`_
+- Fixes #6898 (:pr:`6901`) `Martin Durant`_
```
Comment (jsignell) [inline docs/source/changelog.rst:50]:
```suggestion
- Fix: exception when reading an item with filters (:pr:`6901`) `Martin Durant`_
```

**Review by jsignell — COMMENTED** [review]
These highlights look good to me! There was a fair amount of work on SVD improvements as well.

**Comment by jrbourbeau** [comment]
Thanks for reviewing @jsignell!

**Pull request merged**